packages/server/tests/helpers/prerequisite.ts

```diff
@@ -11,19 +11,11 @@ const RELAY_URL = process.env.E2E_RELAY_HOST || LOCAL_RELAY_URL;
 
 (function () {
   if (USE_LOCAL_NODE) {
-    process.env['NETWORK_NODE_IMAGE_TAG'] = '0.45.0-alpha.0';
-    process.env['HAVEGED_IMAGE_TAG'] = '0.45.0-alpha.0';
-    process.env['MIRROR_IMAGE_TAG'] = '0.92.0';
-
-    console.log(
-      `Docker container versions, services: ${process.env['NETWORK_NODE_IMAGE_TAG']}, mirror: ${process.env['MIRROR_IMAGE_TAG']}`,
-    );
-
```

Comment:
I think there are pros and cons of having the latest images always. we did have a regression lately due to that fact, the good thing is that it helped us to find issues faster and sooner, the bad is that it blocked the CI pipeline for a couple days.

@Nana-EC @ebadiere, What are your thoughts on this?

I personally prefer to get latests the sooner so we catch potencial issues earlier.

Reply:
Same thoughts.
Latest is great but it takes control away from us.
If there's a way to have a back up config whereby if local node had a version different from what we want to test that would be good.
I think there's such a flow

Reply:
If it's blocked we can just add wanted tag versions in the yml file, until issue is resolved.

Reply:
I think let's get this in as noted.
@ebadiere is working on the Configurable Manual Tests that can help with concerns of running targeted tests on specific versions